### PR TITLE
Part 1: Enabling vFile tests on 6.0 ESX

### PR DESCRIPTION
**Pull request by ashahi1**
This PR fixes issue #1947

### CODE REVIEW

**Comment by shuklanirdesh82** [comment]
@ashahi1 your proposed changes look good, you may want to check upon CI failure.

```
FAIL: advanced_vfile_test.go:69: AdvancedVFileTestSuite.TestVFileVolumeLifecycle

advanced_vfile_test.go:76:
    c.Assert(err, IsNil, Commentf(out))
... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc420011ee0), Stderr:[]uint8(nil)} ("exit status 1")
... Error response from daemon: create vfilevolume623934: VolumeDriver.Create: Failed to create volume vfilevolume623934. Reason: Failed to create etcd client
```

**Comment by luomiao** [comment]
This is the same problem we have met with 6.5.

This means the port for ETCD is not open on those 6.0 photon os VMs.
@ashahi1 you need to open them the same way as what we have done at 6.5 side.

File: misc/drone-scripts/setup.sh [hunk 1]
```diff
@@ -13,9 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo "Resetting testbed"
+echo "Resetting testbed!!"
 govc snapshot.revert -vm $ESX_6_5 init
-govc snapshot.revert -vm $ESX_6_0 init
+# will revert the snapshot name to init before merging the changes to master
```
Comment (shuklanirdesh82) [inline misc/drone-scripts/setup.sh]: remove this file from the PR and make sure CI esx is having `init` snapshot correctly set

Comment (ashahi1) [reply misc/drone-scripts/setup.sh]: Done.

**Pull request merged**